Apperently there's a bug in `./crates/burn-cube/src/runtime_tests/cmma.rs` in line 72 when running example "guide"

I cloned the burn repository and simply entered `cargo build --example guide --release`. The project failed to compile: 
```sh
error[E0599]: no method named `read_sync` found for struct `Vec<u8>` in the current scope
  --> crates/burn-cube/src/runtime_tests/cmma.rs:71:45
   |
71 |     let actual = client.read(out.binding()).read_sync().unwrap();
   |                                             ^^^^^^^^^
   |
help: there is a method `read_at` with a similar name, but with different arguments
  --> /rust/deps/object-0.32.2/src/read/read_ref.rs:84:5

For more information about this error, try `rustc --explain E0599`.
error: could not compile `burn-cube` (lib) due to 1 previous error
warning: build failed, waiting for other jobs to finish...
```
I encountered the bug both on Arch Linux (running within WSL on Windows 11) and natively on Windoes 11. 
I also tested another example: "mnist". It doesn't work.

My rust toolchains:
stable-x86_64-unknown-linux-gnu (default) (linux - Arch Linux WSL)
stable-x86_64-pc-windows-gnu (default) (windows 11)

--- a/crates/burn-cube/src/runtime_tests/cmma.rs
+++ b/crates/burn-cube/src/runtime_tests/cmma.rs
@@ -68,7 +68,7 @@ pub fn test_simple_1<R: Runtime>(client: ComputeClient<R::Server, R::Channel>) {
         ArrayHandle::new(&out, 256),
     );
 
-    let actual = client.read(out.binding()).read_sync().unwrap();
+    let actual = client.read(out.binding());
     let actual = f32::from_bytes(&actual);
 
     let expected = [

